.github/workflows/package.yml

```diff
@@ -0,0 +1,86 @@
+name: release
+
+on:
+  push:
+    tags: ["release-*"]
+
+env:
+  BUILD_TYPE: Release
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
```

Comment:
I'm on Debian myself (at work), is Debian stable an alternative as well?

Reply:
Unfortunately the only github-hosted runners available are:

* Ubuntu 20.04 `ubuntu-latest` or `ubuntu-20.04`
* Ubuntu 18.04 `ubuntu-18.04`

We could pin to 18.04, which might better match various package versions.

Using Debian would require a self-hosted runner, which for which one would have to provide appropriately configured infrastructure (e.g., ec2 instance.)